Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -3042,7 +3060,7 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 670,
+          "x": 720,
```

Comment:
Please also add in the prisma tasks - the "skip is automation... unavailable "

Reply:
Done